src/test/java/org/prebid/server/bidder/adtrgtme/AdtrgtmeBidderTest.java

```diff
@@ -0,0 +1,287 @@
+import com.iab.openrtb.request.Site;
+import com.iab.openrtb.request.Video;
+import com.iab.openrtb.response.Bid;
+import com.iab.openrtb.response.BidResponse;
+import com.iab.openrtb.response.SeatBid;
+import io.netty.handler.codec.http.HttpHeaderValues;
+import io.vertx.core.http.HttpMethod;
+import org.junit.Before;
+import org.junit.Test;
+import org.prebid.server.VertxTest;
+import org.prebid.server.bidder.model.BidderBid;
+import org.prebid.server.bidder.model.BidderCall;
+import org.prebid.server.bidder.model.BidderError;
+import org.prebid.server.bidder.model.HttpRequest;
+import org.prebid.server.bidder.model.HttpResponse;
+import org.prebid.server.bidder.model.Result;
+import org.prebid.server.proto.openrtb.ext.ExtPrebid;
+import org.prebid.server.proto.openrtb.ext.request.adrino.ExtImpAdrino;
+import org.prebid.server.util.HttpUtil;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.function.UnaryOperator;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.singletonList;
+import static java.util.function.Function.identity;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.tuple;
+import static org.prebid.server.proto.openrtb.ext.response.BidType.banner;
+
+public class AdtrgtmeBidderTest extends VertxTest {
+
+    private static final String ENDPOINT_URL = "https://z.cdn.adtarget.market/ssp";
+
+    private AdtrgtmeBidder adtrgtmeBidder;
+
+    @Before
+    public void setUp() {
+        adtrgtmeBidder = new AdtrgtmeBidder(ENDPOINT_URL, jacksonMapper);
+    }
+
+    @Test
+    public void makeHttpRequestsShouldReturnHttpRequestWithCorrectBodyHeadersAndMethod() {
+        String siteId = "site_id";
+        // given
+        final BidRequest bidRequest = BidRequest.builder().id("test-request-id").imp(singletonList(
+                givenImp(identity()))).site(Site.builder().id(siteId).build()).build();
+        // when
+        final Result<List<HttpRequest<BidRequest>>> result = adtrgtmeBidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getValue()).hasSize(1).element(0).satisfies(httpRequest -> {
+            assertThat(httpRequest.getMethod()).isEqualTo(HttpMethod.POST);
+            assertThat(httpRequest.getUri()).isEqualTo(String.format("%s?s=%s&prebid", ENDPOINT_URL, siteId));
+            assertThat(httpRequest.getHeaders())
+                    .extracting(Map.Entry::getKey, Map.Entry::getValue)
+                    .containsExactly(
+                            tuple(HttpUtil.CONTENT_TYPE_HEADER.toString(), HttpUtil.APPLICATION_JSON_CONTENT_TYPE),
+                            tuple(HttpUtil.ACCEPT_HEADER.toString(), HttpHeaderValues.APPLICATION_JSON.toString()),
+                            tuple(HttpUtil.X_OPENRTB_VERSION_HEADER.toString(), "2.5"));
+            assertThat(httpRequest.getPayload()).isEqualTo(bidRequest);
+        });
+    }
```

Comment:
Please, split test into multiple smaller ones, so that each test will check only small part of result request, or build HttpRequest<BidRequest> and check with `isEqualTo`.